src/pages/auth/chat/chatList/join/JoinBtn.tsx

```diff
@@ -26,7 +46,9 @@ export default function JoinChatRoom(props: PropsType) {
   };
 
   function joinHandler(e: React.MouseEvent<HTMLButtonElement>) {
```

Comment:
```suggestion
  function joinHandler() {
```
`joinChatRoom`이 제거되어서 event 값은 안 받아도 될것 같습니다

Reply:
넵